sources/rds/db_instance.go

```diff
@@ -109,266 +108,407 @@ func dBInstanceOutputMapper(scope string, _ *rds.DescribeDBInstancesInput, outpu
-				Scope:  scope,
-			}})
+			item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{
+				Query: &sdp.Query{
+					Type:   "rds-db-subnet-group",
+					Method: sdp.QueryMethod_GET,
+					Query:  *dbSubnetGroup,
+					Scope:  scope,
+				},
+				BlastPropagation: &sdp.BlastPropagation{
+					// Changing the subnet group can affect the instance
+					In: true,
+					// The instance won't affect the subnet group
+					Out: false,
+				},
+			})
 		}
 
 		if instance.DBClusterIdentifier != nil {
 			// +overmind:link rds-db-cluster
-			item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{Query: &sdp.Query{
-				Type:   "rds-db-cluster",
-				Method: sdp.QueryMethod_GET,
-				Query:  *instance.DBClusterIdentifier,
-				Scope:  scope,
-			}})
+			item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{
+				Query: &sdp.Query{
+					Type:   "rds-db-cluster",
+					Method: sdp.QueryMethod_GET,
+					Query:  *instance.DBClusterIdentifier,
+					Scope:  scope,
+				},
+				BlastPropagation: &sdp.BlastPropagation{
+					// Tightly coupled
+					In:  true,
+					Out: true,
+				},
+			})
 		}
 
 		if instance.KmsKeyId != nil {
 			// This actually uses the ARN not the id
 			if a, err = sources.ParseARN(*instance.KmsKeyId); err == nil {
 				// +overmind:link kms-key
-				item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{Query: &sdp.Query{
-					Type:   "kms-key",
-					Method: sdp.QueryMethod_SEARCH,
-					Query:  *instance.KmsKeyId,
-					Scope:  sources.FormatScope(a.AccountID, a.Region),
-				}})
+				item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{
+					Query: &sdp.Query{
+						Type:   "kms-key",
+						Method: sdp.QueryMethod_SEARCH,
+						Query:  *instance.KmsKeyId,
+						Scope:  sources.FormatScope(a.AccountID, a.Region),
+					},
+					BlastPropagation: &sdp.BlastPropagation{
+						// Changing the KMS key can affect the instance
+						In: true,
+						// The instance won't affect the KMS key
+						Out: false,
+					},
+				})
 			}
 		}
 
 		if instance.EnhancedMonitoringResourceArn != nil {
 			if a, err = sources.ParseARN(*instance.EnhancedMonitoringResourceArn); err == nil {
 				// +overmind:link logs-log-stream
-				item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{Query: &sdp.Query{
-					Type:   "logs-log-stream",
-					Method: sdp.QueryMethod_SEARCH,
-					Query:  *instance.EnhancedMonitoringResourceArn,
-					Scope:  sources.FormatScope(a.AccountID, a.Region),
-				}})
+				item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{
+					Query: &sdp.Query{
+						Type:   "logs-log-stream",
+						Method: sdp.QueryMethod_SEARCH,
+						Query:  *instance.EnhancedMonitoringResourceArn,
+						Scope:  sources.FormatScope(a.AccountID, a.Region),
+					},
+					BlastPropagation: &sdp.BlastPropagation{
+						// Tightly coupled
+						In:  true,
+						Out: true,
+					},
```

Comment:
How would the monitoring log stream affect the db instance?

Reply:
Maybe if you were to change the log group that a stream was part of, you'd want to know which instance's logs would be affected as part of the blast radius. Not a super string example but I've been tending to err on the side of `true` as you'll see